src/ConstructStyleSheet.js

```diff
@@ -135,7 +119,43 @@ export default class ConstructStyleSheet {
   }
 }
 
+// Implement all methods from the base CSSStyleSheet constructor as
+// a proxy to the raw style element created during construction.
+cssStyleSheetMethods.forEach(method => {
+  ConstructStyleSheet.prototype[method] = function() {
+    if (!sheetMetadataRegistry.has(this)) {
+      throw new Error(
+        `Failed to execute '${method}' on 'CSSStyleSheet': Can't call ${method} on non-constructed CSSStyleSheets.`,
+      )
```

Comment:
Could you give examples of what would throw this error in Chrome?

Reply:
I was just copying existing logic that you already had in code. I didn't write that, just moved it.

Reply:
All of these messages have been updated to "illegal invocation" messages to match Chrome.